src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,66 @@
-export const PeopleFilters = () => {
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { useSearchParams } from 'react-router-dom';
+import { useState } from 'react';
+import { FilterSex } from '../types/Enums';
+
+type Props = {
+  activeSex: string | null;
+  filterQuery: string | null;
+  activeCenturies: string[];
+};
+
+export const PeopleFilters = ({
+  activeSex,
+  filterQuery,
+  activeCenturies,
+}: Props) => {
+  const [filterInputValue, setFilterInputValue] = useState(filterQuery || '');
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const centuries = ['16', '17', '18', '19', '20'];
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = event.target.value || null;
+
+    setFilterInputValue(event.target.value);
+    setSearchParams(getSearchWith(searchParams, { query: newValue }));
+  };
+
+  const toggleCentury = (century: string) => {
+    if (!activeCenturies.includes(century)) {
+      return [...activeCenturies, century];
+    }
+
+    return activeCenturies.filter(activeCentury => activeCentury !== century);
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
+        <SearchLink
+          className={classNames({ 'is-active': activeSex === null })}
+          params={{ sex: null }}
+        >
           All
-        </a>
-        <a className="" href="#/people?sex=m">
+        </SearchLink>
+        <SearchLink
```

Comment:
fix indents for all such cases
```suggestion
        </SearchLink>
        
        <SearchLink
```